### ISSUE TITLE
Display amount of Javascript sent to client

### ISSUE THREAD

**Issue by cbirdsong** [GuerillaHQ/vanillalist#40]
When I am looking for a Javascript library, a primary concern is figuring out how many kilobytes it will add to page weight. It would be great if you could display that information on the page for each library.

(I guess the amount of CSS required would also be useful, but that's generally not as much of a concern as Javascript.)

**Comment by TixieBorg** [comment]
Hi! Thank you for the interest you have in this project, despite the fact its a bit in a dormant state (for now).

That could be indeed a great feature matching the Vanilla List mindset: sharing *lightweight* libraries.

However I'm not sure how I could implement it. Size depend of the way you're importing the lib and which version version you're using. 

I could look at the `main`, `module` and `style` keys from package.json to get the UMD, ESM and style files. But I have no idea if it's commonly used. 
That could be a start at (and be conditionally displayed on the page). 

If you have ideas, you're completely welcome to share them!

**Comment by cbirdsong** [comment]
[Bundlephobia](https://bundlephobia.com) may be a useful resource? In [their repo's readme file](https://github.com/pastelsky/bundlephobia), they imply they have an API but I can't find any info on it.

**Comment by TixieSalander** [comment]
Good idea, I'm keeping in mind when I'll work on it. I have also https://packagephobia.com and https://bundlejs.com as equivalent services
